src/assignment/views/AssignmentOverview.tsx

```diff
@@ -1006,8 +947,12 @@ const AssignmentOverview: FunctionComponent<
 	};
 
 	const renderAssignmentsView = () => {
-		if (!assignments) {
-			return null;
+		if (isLoadingAssignments) {
+			return (
+				<Flex orientation="horizontal" center>
+					<Spinner size="large" />
+				</Flex>
+			);
```

Comment:
Not using `LoadingErrorLoadedComponent`?

Reply:
No, i'm fasing that component out, since it's better handled by react-query functions loading states
and already showing some ui before all the data loads